common/shlibs

```diff
@@ -1013,7 +1013,7 @@ libGeoIP.so.1 libgeoip-1.4.8_1
 libnilfs.so.0 libnilfs-2.1.0_1
 libnilfsgc.so.0 libnilfs-2.1.0_1
 libnilfscleaner.so.0 libnilfs-2.1.0_1
-libchicken.so.8 libchicken-4.11.0_4
+libchicken.so.9 libchicken-5.0.0_3
```

Comment:
this still has the wrong revision

Reply:
Thank you for the helpful comments, I think I finally got it right (hopefully).